site/docs/v1/tech/core-concepts/_refresh-token-settings.adoc

```diff
@@ -3,9 +3,9 @@
 The length of time the refresh token is valid. Refresh tokens are typically long lived.
 
 [field]#Refresh token expiration# [optional]#Optional# [default]#defaults to `Fixed`#::
-The [field]#Refresh token expiration# may be either a fixed or sliding window. By default the expiration of a refresh token is a fixed length of time from when it was originally issued. With a sliding window expiration, the expiration is calculated from the last time the refresh token was used.
+The [field]#Refresh token expiration# may be either a fixed, sliding window, or sliding window with a maximum lifetime. By default, the expiration of a refresh token is a fixed length of time from when it was originally issued. With a sliding window expiration, the expiration is calculated from the last time the refresh token was used. However with a sliding window with a maximum lifetime, the expiration is calculated from the last time the refresh token was used until a maximum lifetime value is reached.
```

Comment:
Either say "may be either fixed, sliding window, or ..." or "may be a fixed window, sliding window, or ...".